src/app/loadout-builder/LoadoutBuilderContainer.tsx

```diff
@@ -87,9 +99,7 @@ export default function LoadoutBuilderContainer({ account }: Props) {
       account={account}
       stores={stores}
       preloadedLoadout={preloadedLoadout}
-      initialClassType={urlClassType}
-      notes={urlNotes ?? preloadedLoadout?.notes}
-      initialLoadoutParameters={urlLoadoutParameters || savedLoadoutParameters}
+      urlParameters={urlParameters}
```

Comment:
The only thing about switching to an object here is that the `memo` in `LoadoutBuilder` won't work anymore.

Reply:
I changed that part back because it honestly didn't make things much clearer.